packages/hooks/useFunnel/useFunnel.tsx

```diff
@@ -41,5 +41,5 @@ export default function useFunnel<S extends string>(steps: [S, ...S[]]) {
     [currentStep]
   );
 
-  return [FunnelComponent, changeStep] as const;
+  return { currentStep, FunnelComponent, changeStep } as const;
```

Comment:
While you're at it, I think adjusting the names like below is more clear and will provide better experience on the user end.

`Funnel --> FunnelComponent`
`FunnelComponent --> FunnelComponentMemo`
`{ FunnelComponent }` --> `{ Funnel: FunnelComponentMemo }`

Reply:
Modification completed👍
[98a1998](https://github.com/fine-ants/demolition/pull/2/commits/98a1998f290bf747a2c1699ffbb631a8a4e2877b)